src/features/stake/components/StakingConfirmationTx/Exit.tsx

```diff
@@ -28,7 +28,7 @@ const StakingConfirmationTxExit = ({ order }: StakingOrderConfirmationViewProps)
             label: 'Exit',
           },
           {
-            value: order.value || '0',
+            value: (order as unknown as { value: string }).value || '0',
```

Comment:
This is needed because the type in the gateway SDK is incorrect, it omits `value` from the TxInfo incorrectly.

Reply:
It works for me with the new SDK like so:

```
import type { components } from '@safe-global/safe-client-gateway-sdk'

type StakingOrderConfirmationViewProps = {
  order: components['schemas']['NativeStakingValidatorsExitConfirmationView']
}
```

Reply:
Migrating to the new SDK is out of scope for this PR. Besides, the type in question isn't `NativeStakingValidatorsExitConfirmationView`, it's the StakingTxInfo.